src/test/java/decodex/modules/hex/HexEncoderTest.java

```diff
@@ -0,0 +1,26 @@
+package decodex.modules.hex;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import decodex.data.Data;
+import org.junit.jupiter.api.Test;
+
+class HexEncoderTest {
+
+    @Test
+    void run_ascii_expectHexEncoding() {
```

Comment:
Perhaps replace `ascii` with `allAsciiChars` to provide more meaning to the input?